gateway/src/apicast/policy/jwt_claim_check/apicast-policy.json

```diff
@@ -0,0 +1,86 @@
+{
+  "$schema": "http://apicast.io/policy-v1/schema#manifest#",
+  "name": "JWT Claim policy",
+  "summary": "Allow or deny traffic based on a JWT claim",
+  "description": [
+    "This Policy allow to block traffic based on a JWT token.",
+    "To verify any JWT claim can be used and can be compared ",
+    "using plain or liquid filters, also whitelist and blaclist."
+  ],
+  "version": "builtin",
+  "configuration": {
+    "type": "object",
+    "properties": {
+      "error_message": {
+        "title": "Error message",
+        "description": "Error message to show to user when traffic is block",
+        "type": "string"
+      },
+      "operations": {
+        "type": "array",
+        "items": {
+          "required": [
+            "match",
+            "match_type",
+            "op",
+            "value"
+          ],
+          "type": "object",
+          "properties": {
+            "match": {
+              "description": "Match operation to get the JWT claim", 
+              "type": "string"
+            },
+            "match_type": {
+              "description": "How to evaluate 'match'",
+              "type": "string",
+              "default": "plain",
+              "oneOf": [
+                {
+                  "enum": ["plain"],
+                  "title": "Evaluate 'value' as plain text."
+                },
+                {
+                  "enum": ["liquid"],
+                  "title": "Evaluate 'value' as liquid."
+                }
+              ]
+            },
+            "op": {
+              "description": "Match operation to compare match JWT claim with provided value", 
+              "type": "string",
+              "enum": [
+                "==",
+                "!=",
+                "matches",
+                ">", 
+                ">=",
+                "<",
+                "<="
+              ]
+            },
+            "value": {
```

Comment:
Maybe we should support evaluating `value` as liquid as well. We do this in policies like the routing one.
More specifically, one of the options available in the routing policy consists of routing based on a jwt claim and the policy allows users to compare it with a value that can be evaluated as liquid. I think this is basically the same use case as here.